src/v2/Apps/Partner/Components/PartnerArtists/PartnerArtistListPlaceholder.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react"
+import {
+  Box,
+  Column,
+  GridColumns,
+  SkeletonBox,
+  SkeletonText,
+} from "@artsy/palette"
+import { PartnerArtistListContainer } from "./PartnerArtistList"
+
+interface PartnerArtistListPlaceholderProps {
+  done?: boolean
+}
+
+export const PartnerArtistListPlaceholder: React.FC<PartnerArtistListPlaceholderProps> = ({
+  done = true,
+}) => (
+  <PartnerArtistListContainer>
+    <GridColumns minWidth={[1100, "auto"]} pr={[2, 0]} gridColumnGap={1}>
+      <Column span={12}>
+        <SkeletonText variant="mediumText" done={done}>
+          Represented Artists
+        </SkeletonText>
+        <Box style={{ columnCount: 6 }} mt={2}>
+          {[...new Array(60)].map((_, i) => {
+            return (
+              <SkeletonBox key={i} mb={1} done={done}>
+                <SkeletonText></SkeletonText>
+              </SkeletonBox>
```

Comment:
Just noticing now that the skeleton components aren't being used quite right. Sorry I missed this.

Basically you want to just use `SkeletonText` here not `SkeletonBox`. [Take a look at the example stories.](https://github.com/artsy/palette/blob/02d894ee5c4487ad3b4782152d42351dcc828eab/packages/palette/src/elements/Skeleton/Skeleton.story.tsx#L23-L59)

You use it exactly like you would `Text` including the `variant` you are anticipating. So in this case maybe some kind of random length text to make it look less brick-like? I'd consult with design about what the placeholder should ideally look like though.

Reply:
Yea, I see. I used empty `SkeletonText` just to get the right item height. Okay, I will ask about placeholder design.

Reply:
Added pseudo-random width for the placeholder items. 

![image](https://user-images.githubusercontent.com/79979820/113891775-11a4c400-97ce-11eb-86b3-73f45389f66e.png)

Reply:
Looks great!